core/src/main/kotlin/org/evomaster/core/problem/graphql/builder/GraphQLActionBuilder.kt

```diff
@@ -376,31 +379,31 @@ object GraphQLActionBuilder {
 
             "int" ->
                 if (element.isKindOfFieldTypeOptional)
-                    OptionalGene(element.fieldName, IntegerGene(element.fieldName))
+                    NullableGene(element.fieldName,OptionalGene(element.fieldName, IntegerGene(element.fieldName)))
```

Comment:
this looks weird... an optional gene inside a nullable one? i would have expected the other way round: ie. something can be optional, and, if there, it could be null

Reply:
note this applies to all cases here

Reply:
Done